Tests/Rules/UseSingularNounsReservedVerbs.tests.ps1

```diff
@@ -15,7 +15,7 @@ BeforeAll {
 }
 
 # UseSingularNouns rule doesn't exist in the non desktop version of PSScriptAnalyzer due to missing .Net Pluralization API
-Describe "UseSingularNouns" -Skip:$IsCoreCLR {
+Describe "UseSingularNouns" {
```

Comment:
we should probably add tests like:
`$result = Invoke-ScriptAnalyzer -ScriptDefinition 'function Get-horses { "horses" }'`
is null, etc.